pagopa/io-app
# fix(Cross): [IOAPPX-395] Open autoresizable bottom sheet on first tap on iOS

## Short description
This PR attempts to fix an issue with the autoresizable bottom sheet on iOS: on this platform the bottom sheet does not open on first tap.

The workaround is to increase the default snap point on iOS: with a bigger value the bottom sheet is opened immediately, but a slight flash remains.

## List of changes proposed in this pull request
- Increased default snap point on iOS

## How to test
Use the bottom sheet screen in the Design System playground.

Please **test as many instances of the autoresizable bottom sheet as possible**, to ensure the workaround covers all cases.

|Before (iOS)|After (iOS)|
|------------|-----------|
|<video width="340" src="https://github.com/user-attachments/assets/16c306c5-ae7b-4802-8566-e1a96eebdcf7"/>|<video width= "340" src="https://github.com/user-attachments/assets/ea28c8ad-3a33-48d9-ac02-9282decbea22"/>|
